.github/workflows/dependabot-auto-merge.yml

```diff
@@ -1,7 +1,7 @@
 # Auto-merge Dependabot PRs. Requires also `.github/.kodiak.toml`.
 # See https://hackmd.io/@SkynetLabs/ryFfInNXc.
 
-name: Dependabot auto-merge
+name: "Approve and Merge Dependabot PRs"
```

Comment:
Now with composite actions this file is pretty small. 

It appears the only reason this is separate is because you only what it to run on pull requests right?

You can do that like this
```yml
if: ${{ github.event_name == 'pull_request' }}
```

Then you can just have one `ci.yml` file. 

See this example: https://github.com/SkynetLabs/ansible-playbooks/pull/286/files#diff-4e726665ead993a5754dee6ff31e7085416505e3a514b09f03f89344166f9f7dR53

Reply:
Do you mean combining the dependabot and npm-checks actions? I was thinking about doing that, you're right that it would be a bit cleaner with less files.

Reply:
yea, if the only reason you are making different files is because of the different triggers, you can just use the if logic to skipp tasks.